extensions/amp-story-dev-tools/0.1/amp-story-dev-tools.js

```diff
@@ -189,26 +197,38 @@ export class AmpStoryDevTools extends AMP.BaseElement {
    * @private
    */
   buildTabs_() {
-    this.tabContents_ = Object.values(DevToolsTab).reduce(
-      (tabContents, tab) => {
-        tabContents[tab] = createTabElement(this.win, this.storyUrl_, tab);
-        return tabContents;
-      },
-      {}
+    const container = this.element.querySelector(
+      '.i-amphtml-story-dev-tools-container'
+    );
+    this.tabContents_[DevToolsTab.PREVIEW] = createTabPreviewElement(
+      this.win,
+      this.storyUrl_,
+      this.hashParams_['devices']
+    );
+    this.tabContents_[DevToolsTab.PAGE_EXPERIENCE] = createTabElement(
+      this.win,
+      this.storyUrl_,
+      DevToolsTab.PAGE_EXPERIENCE
+    );
+    this.tabContents_[DevToolsTab.LOGS] = createTabElement(
+      this.win,
+      this.storyUrl_,
+      DevToolsTab.LOGS
     );
+    Object.values(this.tabContents_).forEach((tabContent) => {
+      container.appendChild(tabContent);
+      toggle(tabContent, false);
```

Comment:
Cf comment on the other PR

Reply:
The 3 tabs get built independently of the order of these 2 lines. Toggle just adds or removes the attribute `hidden` but doesn't seem to affect the build lifecycle. WIll invert the two lines anyways